qiskit/test/mock/utils/configurable_backend_v2.py

```diff
@@ -0,0 +1,183 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Configurable backend for BackendV2."""
+import datetime
+from typing import Optional, List, Tuple, Union, Dict
+
+import numpy as np
+
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.parameter import Parameter
+from qiskit.circuit.measure import Measure
+from qiskit.circuit.reset import Reset
+from qiskit.providers.backend import BackendV2, QubitProperties
+from qiskit.providers.options import Options
+from qiskit.transpiler import Target, InstructionProperties
+
+
+class ConfigurableFakeBackendV2(BackendV2):
+    """Factory for generating mock backendv2 instances"""
+
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        n_qubits: int,
```

Comment:
We've standardized on using `num_qubits` everywhere in the qiskit api (the only exception was the models in `qiskit.providers.models` as those were documented externally as part of the payload specification those are based on). So it would be better to be consistent here and use `num_qubits` instead.